pkg/catalog/types.go

```diff
@@ -36,6 +42,10 @@ func FromConfig(cfg catalog.CatalogConfig) Catalog {
 				LocalUrl:  cfg.DataplaneToken.LocalUrl,
```

Comment:
> Jakub Dyszkiewicz
answering many questions about this dataplaneToken localurl:
previously the behavior was that if token server is disabled then verification is disabled and dataplaneToken.localUrl in catalog is empty and that’s how we could detect that verification is disabled in kuma-dp and kumactl.
now, we cannot turn off the admin server if verification is disabled and serve admin.localUrl empty because even when verification is turned off someone may use provided ca manager anyways.